source/features/repo-avatars.tsx

```diff
@@ -8,31 +7,26 @@ import {getRepo} from '../github-helpers/index.js';
 import getUserAvatar from '../github-helpers/get-user-avatar.js';
 
 async function init(): Promise<void> {
-	// Icon for public but not template/fork/etc. repos
-	const icon = await elementReady('#repository-container-header .octicon-repo');
-	if (!icon) {
-		return;
-	}
-
-	const link = select('#repository-container-header a[rel="author"]')!.cloneNode();
+	// There are many such "label" elements
+	const location = await elementReady('.AppHeader-context-full .AppHeader-context-item-label');
 	const username = getRepo()!.owner;
-	const size = 24;
+	const size = 16;
```

Comment:
Smaller to make it look like an emoji and doesn't compete as much with the site logo.